packages/util/src/markdownHelpers.ts

```diff
@@ -10,9 +10,39 @@
 import { Remarkable } from 'remarkable';
 import parse from 'html-react-parser';
 
-export const parseMarkdown = (embeddedCaption: string) => {
+export type ParseType = 'caption' | 'body';
+
+export const parseMarkdown = (embeddedCaption: string, parser: ParseType = 'caption') => {
   const markdown = new Remarkable({ breaks: true, html: true });
   markdown.inline.ruler.enable(['sub', 'sup']);
+  if (parser === 'caption') {
```

Comment:
Er det ønskelig med breaks og html for captions?

Reply:
Kan fjerne det.